email-templates/201012.html

```diff
@@ -0,0 +1,105 @@
+             sedan kan programmera själv. Anmäl dig på <a href="https://lithekod.se/lodol/">lithekod.se/lodol</a>
+             senast <b class="TODO">DATUM</b>. </li>
+
+        <li> Vårt traditionsenliga Fall Game Jam kommer gå 13-15 november. Som i
+             våras kommer det hållas på distans i <b class="TODO">LiU Game Jams
+             Discordserver (LÄNK)</b> Se till att <b class="TODO">göra något för
+             att bli uppdaterad</b>. </li>
+
+        <li> Höstmötet kommer att hållas i
+             vår <a href="https://discord.gg/2BkvvVx">discordserver</a> tisdag
+             24:e november kl 17.15. Samtliga medlemmar får närvara med
+             rösträtt.
+             Se <a href="https://lithekod.se/posts">https://lithekod.se/posts</a>
+             för mer information. </li>
+      </ul>
+    </p>
+    <p>
+    --<br>
+    LiTHe kods styrelse
+    </p>
+    <p>
+      Om du inte vill få fler informationsutsick från oss, <a
+      href="{unsubscribe_link}">avsluta din prenumeration här</a>.
+    </p>
+
+    <hr>
+
+    <p>
+      <b class="TODO">Någon fräsig ingress</b>
+    </p>
+    <p>
+      <ul>
+        <li> Don't miss Hacktoberfest, an event throughout October celebrating
+             open source software. If you submit four pull requests to
+             (participating) projects on Github you'll get a Hacktoberfest
+             T-shirt!  (You can also choose to plant a tree.) Read more at
+             <a href="http://hacktoberfest.com">http://hacktoberfest.com</a> </li>
+
+        <li> On the 7th of November it is once again time for NCPC! NCPC is a
+             day-long programming contest where teams of up to three persons
+             together try to solve programmering problems of various difficulty.
+             The best team <b class="TODO">teams?</b> will advance to NWERC
+             (Northwestern Europe Regional Contest). NCPC is usually conducted
+             on-site only but this year you can take part both on-site and from
+             home.
+             See <a href="https://lithekod.se/ncpc/en">https://lithekod.se/ncpc</a>
+             for more information and how to sign up for the competition. </li>
+
+        <li> Do you want to build an Arduino, solder and drink beer (or
+             something else to drink with or without alcohol)? In that case,
+             join our Lödöl together with MindRoad the 12th of November! You'll
+             get the change to solder an Arduino, a display and a couple of
+             sensors to a circuit board that you can program yourself. Sign up
+             via <a href="https://lithekod.se/lodol/">lithekod.se/lodol</a> before <b class="TODO">DATE</b>. </li>
+
+        <li> Our Fall Game Jam will this year be held between the 13th and 15th
+             of November. Like in spring, it will be kept in <b class="TODO">LiU
+             Game Jam's Discord server (LINK)</b>. Make sure to <b class="TODO">
+             visit our website? Or something</b> </li>
```

Comment:
```suggestion
             of November. Like in spring, it will be kept in <a href="https://discord.gg/xwXAgyE">LiU
             Game Jam's Discord server</a>. Make sure to <a href="liugamejam.se">
             visit liugamejam.se</a> for some radical tips! </li>
```